pkgs/development/tools/build-managers/xmake/default.nix

```
@@ -1,48 +1,29 @@
-{ lib
-, stdenv
-, fetchurl
-, pkg-config
-, lua
-, readline
-, ncurses
-, lz4
-, tbox
-, xmake-core-sv
+{
+  lib,
+  stdenv,
+  fetchurl,
+  CoreServices,
+  ...
 }:
-
 stdenv.mkDerivation rec {
   pname = "xmake";
-  version = "2.9.4";
-
+  version = "v2.9.5";
```

```suggestion
  version = "2.9.5";
```
The 'v' prefix should not be in the `version`

---

Which version are you reviewing?

https://github.com/rennsax/nixpkgs/blob/1fe3e73b85eb5a16e2111dbf78b621439f0b1f2b/pkgs/development/tools/build-managers/xmake/default.nix#L1-L5